src/SFA.DAS.Tools.Support.Web/Views/Support/Index.cshtml

```diff
@@ -1,41 +1,49 @@
-﻿@{
+﻿@using Microsoft.AspNetCore.Authorization
+@using SFA.DAS.Tools.Support.Web.Infrastructure
+
+@inject IAuthorizationService AuthorizationService
+@{
     @using SFA.DAS.Tools.Support.Web.Configuration;
     ViewData["Title"] = "Support Portal";
+    var result = AuthorizationService.AuthorizeAsync(User, nameof(PolicyNames.HasTier3Account)).Result.Succeeded;
```

Comment:
can we handle this in the controller and pass the information in via the viewmodel please?